census_area/__init__.py

```diff
@@ -89,7 +91,14 @@ def _state_place_area(self, method, fields, state, place, year=None, return_geom
                                            extra_query_args={'where': search_query,
                                                              'orderByFields': 'OID'})
 
-        place = next(iter(place_dumper))
+        try:
+            place = next(iter(place_dumper))
+        except TypeError as e:
+            if "'<' not supported between instances of 'NoneType' and 'NoneType'" in str(e):
+                raise ValueError(f'Could not find specified place "{place}" in state "{state}"')
```

Comment:
Open to other interpretations of this error, if you have any ideas @fgregg.

Reply:
make sense. let's bring this in and try to fix upstream so it throws a better error